patches/net.minecraft.client.settings.GameSettings.java.patch

```diff
@@ -35,30 +35,24 @@
     private static final String[] AMBIENT_OCCLUSIONS = new String[]{"options.ao.off", "options.ao.min", "options.ao.max"};
     private static final String[] STREAM_COMPRESSIONS = new String[]{"options.stream.compression.low", "options.stream.compression.medium", "options.stream.compression.high"};
     private static final String[] STREAM_CHAT_MODES = new String[]{"options.stream.chat.enabled.streaming", "options.stream.chat.enabled.always", "options.stream.chat.enabled.never"};
-@@ -169,7 +169,7 @@
-    public String language;
-    public boolean forceUnicodeFont;
- 
--   public GameSettings(Minecraft mcIn, File optionsFileIn) {
-+   public GameSettings(Minecraft mcIn, File optionsFileIn) throws IOException {
-       this.keyBindings = (KeyBinding[])ArrayUtils.addAll(new KeyBinding[]{this.keyBindAttack, this.keyBindUseItem, this.keyBindForward, this.keyBindLeft, this.keyBindBack, this.keyBindRight, this.keyBindJump, this.keyBindSneak, this.keyBindSprint, this.keyBindDrop, this.keyBindInventory, this.keyBindChat, this.keyBindPlayerList, this.keyBindPickBlock, this.keyBindCommand, this.keyBindScreenshot, this.keyBindTogglePerspective, this.keyBindSmoothCamera, this.keyBindStreamStartStop, this.keyBindStreamPauseUnpause, this.keyBindStreamCommercials, this.keyBindStreamToggleMic, this.keyBindFullscreen, this.keyBindSpectatorOutlines}, this.keyBindsHotbar);
-       this.difficulty = EnumDifficulty.NORMAL;
-       this.lastServer = "";
-@@ -177,7 +177,12 @@
+@@ -177,7 +177,15 @@
        this.language = "en_US";
        this.forceUnicodeFont = false;
        this.mc = mcIn;
 -      this.optionsFile = new File(optionsFileIn, "options.txt");
 +      File def = new File(optionsFileIn, "options.txt");
 +      this.optionsFile = new File(optionsFileIn, "options-hyperium.txt");
 +      if(!optionsFile.exists()) {
-+         Files.copy(def.toPath(), optionsFile.toPath(), StandardCopyOption.REPLACE_EXISTING);
++         try {
++            Files.copy(def.toPath(), optionsFile.toPath());
++         } catch (IOException e) {
```

Comment:
never thrown, gonna end up renaming the variable to ignored rather than e to avoid warnings